conans/server/service/common/search.py

```diff
@@ -61,7 +60,8 @@ def search_packages(server_store, ref, query, look_in_all_rrevs):
     if not os.path.exists(server_store.conan_revisions_root(ref.copy_clear_rev())):
         raise RecipeNotFoundException(ref)
     infos = _get_local_infos_min(server_store, ref, look_in_all_rrevs)
-    return filter_packages(query, infos)
+    assert query is None, "The server is not filtering packages remotely anymore"
```

Comment:
Maybe add a version client message necessary, because this might crash for older Conan clients?

Reply:
But very very old. We can remove the assert if you want.

Reply:
It is fine, lets leave it.